encoding/csv/mod.ts

```diff
@@ -0,0 +1,139 @@
+import { BufReader, BufState } from "../../io/bufio.ts";
```

Comment:
Rename file to `encoding/csv.ts`

Please add 
```
// Ported from Go: Copyright 2009 The Go Authors.
// https://github.com/golang/go/blob/go1.12.5/src/encoding/csv/csv.go
// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
```

Reply:
Will surely do.